src/components/InactiveClientsList.js

```diff
@@ -0,0 +1,103 @@
+
+    const handleChange = (event) => {
+        setState({ ...state, [event.target.name]: event.target.checked })
+    }
+
+    const [state, setState] = React.useState({
+        checkedA: false,
+    });
+    const { loading: loadingInactive , error: errorInactive, data: dataInactive } = useQuery(GET_INACTIVE_CLIENTS_COUNT);
+
+    if (loadingInactive) return <LoadingProgress/>
+    if (errorInactive) return `Error! ${errorInactive.message}`;
+
+    const renderClientTiles = (clients) => {
+        return clients.map(c => {
+            return (
+                <Grid item xs={12} sm={6} lg={4}>
+                    <ClientTile
+                        client={c}
+                        history={history}
+                    />
+                </Grid>
+            )
+        })
+    }
+
+    if (loading) return <LoadingProgress/>
+    if (error) return `Error! ${error.message}`;
+    const clients = orderBy(data.getInactiveClients, 'is_active', 'desc')
+
+    return (
+        <>
+            <FormGroup>
+                <FormControlLabel
+                    control=
+                        {
+                            <Switch
+                                checked={state.checkedA}
+                                onChange={handleChange}
+                                name='checkedA'
+                            />
+                        }
+                    label='Show inactive clients'
+                />
+            </FormGroup>
+            <Grid
+                container
+                direction='row'
+                justify='space-between'
+                alignItems='flex-end'
+            >
+                {
+                    state.checkedA == true
+                        ? (
+                            <Grid item xs={12} sm={6} md={4}>
+                                <Box
+                                    bgcolor='primary.black'
+                                    color='primary.light'
+                                    borderRadius='borderRadius'
+                                    px={2}
+                                    px-lg={5}
+                                    py={1}
+                                >
+                                    {
+                                        `${dataInactive.getInactiveClientsCount} inactive ${dataInactive.getInactiveClientsCount == 1
+                                            ? 'client'
+                                            : 'clients'
+                                        }`
+                                    }
+                                </Box>
+                            </Grid>
+                        ) : (
+                            false
+                        )
+                }
+                <Grid container>
+                    {
+                        state.checkedA ? clients.length != 0 ? renderClientTiles(clients) : false : false
+                    }
```

Comment:
Nested ternaries like this are always tricky to make sure they are not confusing to the reader, I recommend:

```
{
    state.checkedA
        ? clients.length != 0
                ? renderClientTiles(clients)
                : null
        : null
}
```

Note you should use `null` instead of `false` to render nothing in React